WebPanel/pages/clients/index.tsx

```diff
@@ -76,16 +109,16 @@ async function updateClientsAsync(params: GridCellEditCommitParams): Promise<Cli
 async function deleteClientsAsync(clientId: number): Promise<ClientAPIResponse<DeleteClientAPIResponse>> {
     const resp = await fetch('/api/clients', {
         method: 'DELETE',
-        headers: { 'Content-Type': 'application/json' },
-        body: JSON.stringify({ id: clientId })
+        headers: {'Content-Type': 'application/json'},
+        body: JSON.stringify({id: clientId})
```

Comment:
again, I do prefer spaces in the object, is there a standard of not doing so, or am I abnormal.

Reply:
done